app/models/manageiq/providers/openshift/inventory/parser/container_manager.rb

```diff
@@ -0,0 +1,43 @@
+class ManageIQ::Providers::Openshift::Inventory::Parser::ContainerManager < ManageIQ::Providers::Kubernetes::Inventory::Parser::ContainerManager
```

Comment:
I'm confused now.  Why is this a new file, how does this relate to the existing parser?
(same question for the k8s PR, now that I think of it)

Reply:
@Ladas and I were talking about wanting to standardize on graph refresh and cleanup the existing parser since it is a mix of normal and graph refresh.  Eventually we would move it all out into this parser and drop the option for normal refresh but we don't want to impact existing code yet.

Reply:
I'm all for dropping classic refresh ✂️ :)